Tools/packages.config

```diff
@@ -7,7 +7,6 @@
   <package id="NUnit.Extension.NUnitV2ResultWriter" version="3.6.0" targetFramework="net461" />
   <package id="NUnit.Extension.TeamCityEventListener" version="1.0.6" targetFramework="net461" />
   <package id="NUnit.Extension.VSProjectLoader" version="3.8.0" targetFramework="net461" />
-  <package id="CSharpAsyncGenerator.CommandLine" version="0.14.0" targetFramework="net461" />
   <package id="vswhere" version="2.1.4" targetFramework="net461" />
```

Comment:
`vswhere` is used by [actual_msbuild.cmd](https://github.com/nhibernate/nhibernate-core/blob/master/Tools/actual_msbuild.cmd#L3), used for generating the documentation, see [here](https://github.com/nhibernate/nhibernate-core/blob/master/doc/documentation.build#L68).

Reply:
Thanks, now it is moved to `packages.csproj`.